ArgosIOS/Argos/Views/LandingPage/LandingViewModel.swift

```diff
@@ -21,6 +21,9 @@ class LandingViewModel: LoadableObject {
     @Published var packTemp: Double = 0
     @Published var motorTemp: Double = 0
     
+    @Published var latitude: Double = 42.34037221430655
+    @Published var longitude: Double = -71.09065805832445
```

Comment:
where is this? in boston

Reply:
this is wollastons lol i thought it would be good default for now